org.scala-ide.sdt.core.tests/src/org/scalaide/core/sbtbuilder/NestedProjectsTest.scala

```diff
@@ -95,7 +95,8 @@ class NestedProjectsTest {
       assertEquals("No errors in top-level project", 0, topLevelErrors.length)
 
       val nestedErrors = scalaProject.underlying.findMarkers(IJavaModelMarker.JAVA_MODEL_PROBLEM_MARKER, true, IResource.DEPTH_INFINITE)
-      assertEquals("One error in nested project", 1, nestedErrors.length)
+      val errors = SDTTestUtils.userFriendlyMarkers(nestedErrors.toList)
+      assertEquals("One error in nested project " + errors, 1, errors.length)
```

Comment:
Could we reuse this error handling logic in both this files (this and ClasspathTests.scala)?